### PR TITLE
docs: Add WalletDropdownDisconnect and WalletDropdownLink page

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**

WalletDropdownLink:

Screenshot 2024-08-14 at 1 40 04 PM

Screenshot 2024-08-13 at 11 28 07 AM

Screenshot 2024-08-13 at 11 28 24 AM

Screenshot 2024-08-13 at 11 28 40 AM

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/wallet/wallet-dropdown-disconnect.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  WalletDropdownBaseName,
+  WalletDropdownDisconnect,
+} from '@coinbase/onchainkit/wallet';
+import WalletComponents from '../../components/WalletComponents';
+
+# `<WalletDropdownDisconnect />`
+
+The `WalletDropdownDisconnect` component is used to disconnect the wallet from the application.
+
+## Usage
```
Comment (Zizzamia) [inline site/docs/pages/wallet/wallet-dropdown-disconnect.mdx:23]: For every component is important to teach each props, how to use it.

So in this case we should teach how to use:
- className
- text

File: site/docs/pages/wallet/wallet-dropdown-link.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  WalletDropdown,
+  WalletDropdownBaseName,
+  WalletDropdownDisconnect,
+  WalletDropdownLink,
+} from '@coinbase/onchainkit/wallet';
+import WalletComponents from '../../components/WalletComponents';
+
+# `<WalletDropdownLink />`
+
+The `WalletDropdownLink` creates a custom link within the dropdown. Use the icon prop to add an icon, and href to specify the destination.
```
Comment (Zizzamia) [inline site/docs/pages/wallet/wallet-dropdown-link.mdx:21]: For every component is important to teach each props, how to use it.

So in this case we should teach how to use:
- children
-  className
- href
- icon
- rel
- target

I leave it to you decide how best mix and match the demo.

File: site/sidebar.ts [hunk 3]
```diff
@@ -131,14 +131,22 @@ export const sidebar = [
...
+          },
+          {
+            text: 'WalletDropdownDisconnect',
+            link: '/wallet/wallet-dropdown-disconnect',
+          },
+          {
+            text: 'WalletDropdownLink',
+            link: '/wallet/wallet-dropdown-link',
+          },
           {
```
Comment (Zizzamia) [inline site/sidebar.ts:146]: Let's have Wallet as the first component.

Comment (cpcramer) [reply site/sidebar.ts:146]: Ok - And I will update `Identity` to also be the first component so every section matches. 
Screenshot 2024-08-13 at 3 31 37 PM

File: site/docs/pages/wallet/wallet-dropdown-disconnect.mdx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+</WalletComponents>
+
+### Override text
+
+You can override component text using `text`.
+
+```tsx
+// omitted for brevity
+
+  <WalletDropdownDisconnect text="Log out" /> // [!code focus]
```
Comment (Zizzamia) [inline site/docs/pages/wallet/wallet-dropdown-disconnect.mdx:117]: Why this extra space?
this
```
 <WalletDropdownDisconnect text="Log out" />
```
should be
```
<WalletDropdownDisconnect text="Log out" />
```

those are little details, that usually don't matters, but it's in the little details that we keep things super neat.

File: site/docs/pages/wallet/wallet-dropdown-link.mdx [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+  </Wallet>
+</WalletComponents>
+
+### Custom children components
+Accepts React children, enabling the use of custom elements, styled text, icons, and complex components.
+This allows for diverse customizations, including complex layouts and conditional rendering based on your app's state.
+
+```tsx
+// omitted for brevity
+<WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com"> // [!code focus]
```
Comment (Zizzamia) [inline site/docs/pages/wallet/wallet-dropdown-link.mdx:135]: Follow up for next PR:
- teach how to use custom icons react components. And not just the default `wallet` one.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**